submissions/models.py

```diff
@@ -9,7 +9,7 @@
     ./manage.py makemigrations submissions
 
 """
-
+# pylint: skip-file
```

Comment:
why do we need to skip the entire file instead of fixing/suppressing individual quality warnings

Reply:
I was facing an attribute error here and its due to the new version of pylint